Lombiq.HelpfulLibraries/Lombiq.HelpfulLibraries.csproj

```diff
@@ -1,28 +1,29 @@
 <Project Sdk="Microsoft.NET.Sdk.Razor">
 
-  <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
-    <DefaultItemExcludes>$(DefaultItemExcludes);.git*;node_modules\**</DefaultItemExcludes>
-  </PropertyGroup>
+    <PropertyGroup>
+        <TargetFramework>netcoreapp3.1</TargetFramework>
+        <DefaultItemExcludes>$(DefaultItemExcludes);.git*;node_modules\**</DefaultItemExcludes>
+        <LangVersion>Latest</LangVersion>

```

Comment:
This is better managed on a solution-level, please remove it.

Reply:
How do we manage it at the solution level?

Reply:
Like this: https://github.com/Lombiq/.NET-Analyzers/blob/dev/Build.props#L8 Also, I recommend using a specific language level (as indicated there) so it's the same in all environments, not automatically using the latest.